<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_AddEvent</class>
 <widget class="QDialog" name="Dialog_AddEvent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_day">
     <property name="text">
      <string>روز</string>
     </property>
     <property name="buddy">
      <cstring>comboBox_day</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox_day">
     <item>
      <property name="text">
       <string>شنبه</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>یکشنبه</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>دوشنبه</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>سه شنبه</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>چهارشنبه</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>پنجشنبه</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>جمعه</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_title">
     <property name="text">
      <string>موضوع</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_title</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_title"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_description">
     <property name="text">
      <string>توضیحات</string>
     </property>
     <property name="buddy">
      <cstring>lineEdit_description</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_description"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_AddEvent</receiver>
   <slot>onAcceptClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_AddEvent</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onAcceptClicked()</slot>
 </slots>
</ui>
